runtime/near-vm-runner/src/tests/test_builder.rs

```diff
@@ -29,10 +29,10 @@ pub(crate) fn test_builder() -> TestBuilder {
         output_data_receivers: vec![],
     };
     let mut skip = HashSet::new();
-    if cfg!(not(target_arch = "x86_64")) {
-        skip.insert(VMKind::Wasmer0);
-        skip.insert(VMKind::Wasmer2);
-        skip.insert(VMKind::NearVm);
+    for kind in [VMKind::Wasmer0, VMKind::Wasmer2, VMKind::NearVm, VMKind::Wasmtime] {
```

Comment:
Nit: could we move that list to a static function of `VMKind`, to keep it closer to the definition?

Reply:
Oh right, I thought it was defined in near-vm-runner, but it’s actually in parameters. Makes sense :)